conans/test/integration/generators/json_test.py

```diff
@@ -30,7 +30,7 @@ def package_info(self):
         conan_json = client.load("conanbuildinfo.json")
         data = json.loads(conan_json)
         self.assertEqual(data["dependencies"][0]["version"], "0.1")
-        self.assertIsNone(data["dependencies"][0]["description"])
+        self.assertEqual(data["dependencies"][0]["description"], "my desc")
```

Comment:
I don't know if I'm missing something but we weren't saving the description before those changes.